### ISSUE TITLE
Minimal snips to run CodeT5 on each task

### ISSUE THREAD

**Issue by rocco-fortuna** [salesforce/CodeT5#73]
Hello, I have spent quite some time trying to run the fine-tuned model you kindly provided, on the clone detection task, with no luck.
Could you provide minimal python scripts to load and run the model on each of the downstream tasks, as was done for ntp and summarization on 🤗?

Thanks a lot!

**Comment by rocco-fortuna** [comment]
I am happy to contribute by providing such scripts, but need guidance to make your model work. Your `finetuned_models_clone_codet5_base.bin` model only ever predicts code pairs in the test set are not clones for me, which yields 0 F1 score, not quite the 97% it should reach according to the paper.

**Comment by yuewang-cuhk** [comment]
Hi, we've shared the instructions to reproduce the results using our released checkpoints [here](https://github.com/salesforce/CodeT5#how-to-reproduce-the-results-using-the-released-finetuned-checkpoints). Please try it or you can rerun the finetuning experiments. If both options still do not work, please share more detailed logs for us to understand the issue.
